compute_sdk/globus_compute_sdk/__init__.py

```diff
@@ -18,4 +19,5 @@
     "ContainerSpec",
     "ShellFunction",
     "ShellResult",
+    "MPIFunction",
```

Comment:
`Executor` notwithstanding, we usually order (alphabetize) these, to aid readability and git churn.

Reply:
It looks weird to have this partially ordered. Would you be open to having this list sorted completely?

Reply:
Of course!  _Please_ do that.  :wink: